client/van_site_config_inspect.go

```diff
@@ -25,11 +25,17 @@ func (cli *VanClient) VanSiteConfigInspect(ctx context.Context, input *corev1.Co
 		siteConfig = input
 	}
 	var result types.VanSiteConfig
+	// TODO: what should the defaults be for name, namespace
 	if skupperName, ok := siteConfig.Data["name"]; ok {
 		result.Spec.SkupperName = skupperName
 	} else {
 		result.Spec.SkupperName = cli.Namespace
 	}
+	if skupperNamespace, ok := siteConfig.Data["namespace"]; ok {
+		result.Spec.SkupperNamespace = skupperNamespace
+	} else {
+		result.Spec.SkupperNamespace = cli.Namespace
```

Comment:
here you "Get" a resource, if a field is empty, the client is assuming that the content of the field is his own configured Namespace attribute ... is it possible to  just always set the siteConfig.Data["namespace"] when creating the config map? Same for skupperName ...

Reply:
I think 'namespace' probably should not be an entry in the data section of the configmap, but should be implied by the namespace that configmap is in.

Reply:
+1, will change